### DISCUSSION TITLE
AlpineJS drop down menu to close when I navigate away in BarbaJS

### DISCUSSION THREAD

**Discussion by DESIGNfromWITHIN** [alpinejs/alpine#864] [4. General]
I am using BarbaJS together with Tailwind UI (using AlpineJS)

The menu and header are outside of the data-barba="container" I understand that stuff like active classes should be managed with JS.

But how do I trigger the AlpineJS drop down menu to close when I navigate away?

It uses code like:
x-data=" mobileMenuOpen: false
and
@click="flyoutMenuOpen = !flyoutMenuOpen

**Comment by KevinBatdorf** [comment]
Did you try `@click.away="flyoutMenuOpen = false"` already?

**Reply by DESIGNfromWITHIN** [reply]
Yes it was there but on wrong element... Many thanks!
